cmd/outputs.go

```diff
@@ -63,15 +64,9 @@ func getAllOutputConstructors() (map[string]func(output.Params) (output.Output,
 			return newCollectorAdapter(params, influxc), nil
 		},
 		"kafka": func(params output.Params) (output.Output, error) {
-			conf, err := kafka.GetConsolidatedConfig(params.JSONConfig, params.Environment, params.ConfigArgument)
-			if err != nil {
-				return nil, err
-			}
-			kafkac, err := kafka.New(params.Logger, conf)
-			if err != nil {
-				return nil, err
-			}
-			return newCollectorAdapter(params, kafkac), nil
+			params.Logger.Warn("The kafka output is deprecated, and will be removed in a future version. " +
+				"Please use the xk6 extension that can be found at https://github.com/k6io/xk6-output-kafka.")
```

Comment:
```suggestion
			params.Logger.Warn("The kafka output is deprecated, and will be removed in a future k6 version. " +
				"Please use the new xk6 kafka output extension instead. It can be found at https://github.com/k6io/xk6-output-kafka.")
```